css-components/button.css

```diff
@@ -0,0 +1,74 @@
+:root {
+    --btn-color-primary: #0284C7;
+    --btn-color-secondary: #16A34A ;
+    --btn-color-success: #8ebd4c;
+    --btn-color-cancel: #E13434;
+    --btn-color-warning:#F1B24A;
+    --btn-color-light: white;
+    --btn-color-dark: black;
+    --btn-radius-small: .5rem;
+    --btn-border-width: .12rem;
+}
+
+.btn {
+    text-decoration: none;
+    border: none;
+    padding: .5rem 1rem;
+    margin: .5rem;
+    border-radius: var(--btn-radius-small);
+}
+
+.btn-primary {
+    background-color: var(--btn-color-primary);
+    color:var(--btn-color-light);
+    border: var(--btn-border-width) solid var(--btn-color-primary);
+}
+
+.btn-secondary {
+    background-color: var(--btn-color-secondary);
+    color: var(--btn-color-light);
+    border: var(--btn-border-width) solid var(--btn-color-secondary);
+}
+
+.btn-success {
+    background-color: var(--btn-color-success);
+    color: var(--bnt-color-dark);
+    border: var(--btn-border-width) solid var(--btn-color-success);
+}
+
+.btn-cancel {
+    background-color: var(--btn-color-cancel);
+    border: var(--btn-border-width) solid var(--btn-color-cancel );
+    color: var(--btn-color-light);
+}
+.btn-warning{
+    background-color: var(--btn-color-warning);
+    border: var(--btn-border-width) solid var(--btn-color-warning);
+}
+.btn-dark{
+    background-color: var(--btn-color-dark);
+    border: var(--btn-border-width) solid var(--btn-color-dark);
+    color: var(--btn-color-light);
+}
+.btn-light{
+    /* background-color: white; */
+    border: var(--btn-border-width) solid var(--btn-color-dark);
+    color: var(--btn-color-dark);
+}
+.link {
+    text-decoration: none;
+    box-sizing: border-box;
+    padding: .5rem 1rem;
+    border-radius: var(--btn-radius-small);
+    border: var(--btn-border-width) solid var(--main-color);
+    margin: .5rem;
+}
+.link-primary{
+    background-color: var(--main-color);
+    color: var(--btn-color-light);
+    
+}
+.link-secondary{
+    color: var(--main-color);  
```

Comment:
For link buttons you can check how Chakra has made their buttons, 
[chakra buttons](https://chakra-ui.com/docs/form/button#button-variants)